runtime/src/main/java/org/corfudb/runtime/object/transactions/OptimisticTransactionalContext.java

```diff
@@ -56,6 +49,13 @@
     private final Set<ICorfuSMRProxyInternal> modifiedProxies =
             new HashSet<>();
 
+    /** Keep the version (at global snapshot time)
```

Comment:
@no2chem , the proxy is used here only as the identifier of a stream, it is the key to the hashmap.